src/object.rs

```diff
@@ -1541,7 +1541,7 @@ impl<T: ObjectType> ObjectExt for T {
         ::signal::connect_raw(
             self.as_object_ref().to_glib_none().0,
             signal_name.as_ptr() as *const _,
-            Some(*(&notify_trampoline::<Self, F> as *const _ as *const _)),
+            Some(mem::transmute(notify_trampoline::<Self, F> as usize)),
```

Comment:
```suggestion
            Some(mem::transmute::<_, unsafe extern "C" fn()>(notify_trampoline::<Self, F> as *const ())),
```

etc. Let's not introduce unbounded transmutes again.